src/styles/components/advanced-section/styles.less

```diff
@@ -15,6 +15,10 @@
       }
     }
   }
+
+  .runtimeLabel-badge {
+    margin-left: 7px;
```

Comment:
I really dislike these kind of style additions because:
1. They're small, but they add up and bloat the CSS we're delivering to the browser
2. It takes the engineer extra time to look up an example to base the spacing value off of
3. If the design team ever makes changes to typographic scale or spacing rules to change the visual density of the UI, it will be very painful to implement consistently

This is the kind of thing a CSS utility library would help with. A reusable class like `.marginLeft--small` could have been used here

</rant>

Reply:
Should we open a jira for creating reusable classes?

Reply:
@GeorgiSTodorov - yes, I'll create an epic